### ISSUE TITLE
do themeable browser have plan to implement load progress function

### ISSUE THREAD

**Issue by tomsean** [initialxy/cordova-plugin-themeablebrowser#42]

**Comment by initialxy** [comment]
@tomsean I see your [pull request](https://github.com/initialxy/cordova-plugin-themeablebrowser/pull/43). So yes!

**Comment by anilk533** [comment]
Hi How add loader or prograssbar in themebrowser i want to show  laoder in themebrowser untill its open www.google.com.for now its taking some time to open and untill it opens its showing empty white screen.

this is my code (ima using ionic)





$scope.themebrowser = function()
	 {
		          
						var ref = cordova.ThemeableBrowser.open('http://www.eenadu.net/andhra-pradesh-news.aspx', '_blank',
						{
							statusbar: {
							color: '#ffffffff'
						},
						toolbar: {
							height: 44,
							color: '#f0f0f0ff'
						},
						title: {
							color: '#003264ff',
							showPageTitle: true
						},
						backButton: {
							image: 'back',
							imagePressed: 'back_pressed',
							align: 'left',
							event: 'backPressed'
						},
						forwardButton: {
							image: 'forward',
							imagePressed: 'forward_pressed',
							align: 'left',
							event: 'forwardPressed'
						},
						closeButton: {
							image: 'close',
							imagePressed: 'close_pressed',
							align: 'right',
							event: 'closePressed'
						}
						/*,
						customButtons: [
						{
						image: 'share',
						imagePressed: 'share_pressed',
						align: 'right',
						event: 'sharePressed'
						}
						],
						menu: {
						image: 'menu',
						imagePressed: 'menu_pressed',
						title: 'Test',
						cancel: 'Cancel',
						align: 'right',
						items: [
						{
						event: 'helloPressed',
						label: 'Hello World!'
						},
						{
						event: 'testPressed',
						label: 'Test!'
						}
						]
						}*/
						,
						"browserProgress": {
						"showProgress": true,
						"progressBgColor": "#016585",
						"progressColor": "#FFAA16"
						},
						backButtonCanClose: true,
						fullscreen:true
						}).addEventListener('backPressed', function(e) {
							$ionicHistory.goBack();
						//alert('back pressed');
						}).addEventListener('helloPressed', function(e) {
						//alert('hello pressed');
						}).addEventListener('sharePressed', function(e) {
						//alert(e.url);
						}).addEventListener(cordova.ThemeableBrowser.EVT_ERR, function(e) {
							console.error(e.message);
						}).addEventListener(cordova.ThemeableBrowser.EVT_WRN, function(e) {
						//alert(e.message);
						});
						
						
						
						ref.addEventListener('loadstart', function(event) {
							console.log('loadstart');
							$ionicLoading.show({
							content: 'Loading',
							animation: 'fade-in',
							showBackdrop: true,
							maxWidth: 200,
							showDelay: 0
						  });
						//$scope.progressbar = ngProgressFactory.createInstance();
						//$scope.progressbar.start();
						});
						ref.addEventListener('loadstop', function(event) {
							$ionicLoading.hide();
						//alert('loadstop');
						//$scope.progressbar.stop();
						$ionicLoading.hide();
						});
						ref.addEventListener('event_getURL', function(event) {
						//alert('event_getURL');
						});
						ref.addEventListener('exit', function(event) {
						//alert('exit');
						});
						
						
		 
	 }



if add loader in on event load 'start' its not showing laoder in themble Browser its showing in my actuall view(before page)

how solve this please help me

**Comment by tomsean** [comment]
do you have implement loadstart event?

**Comment by anilk533** [comment]
Yes i implemented but that laoding not showing in this themebrowser its showing in page only(i mean previous page) its not showing loader in themebrowser
